Gh-481: Document nested Named Operations (#482)
gchq/gaffer-doc: docs/administration-guide/named-operations.md

```
For more examples of Named Operations, please refer to the [Named Operations page in the Reference Guide](../reference/operations-guide/named.md).

### Nested Named Operations

Nested Named Operations allow users to run stored Named Operations within one another. 

> To enable this feature users must set the [store property](../administration-guide/gaffer-stores/store-guide.md#all-general-store-properties)
> `gaffer.named.operation.nested` to true.

Reviewer: ```suggestion
This is disabled by default. To enable this feature the [store property](../administration-guide/gaffer-stores/store-guide.md#all-general-store-properties)
`gaffer.named.operation.nested` must be set to true.
```

> There is a [default depth limit](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/graph/hook/NamedOperationResolver.html#DEPTH_LIMIT_DEFAULT) 
> set on nested Named Operations of 3.

Reviewer: ```suggestion
The [default depth limit](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/graph/hook/NamedOperationResolver.html#DEPTH_LIMIT_DEFAULT) 
for nested Named Operations is 3.
```
I'm not sure that we need a link to the code, as looking at this doesn't really give an admin any useful information.

This default can be changed through configuring the NamedOperationResolver [graph hook](../development-guide/project-structure/components/graph.md#graph-hooks).
Using this graph hook you can set the `depthLimit` in the JSON file to whatever you like. 

??? example "Example Nested Named Operation"

>     Create a simple Named Operation, then nest this inside another Named Operation. 

Reviewer: Might be good to mention somewhere that the operations need to be created in an order such that they never refer to operations which don't exist yet. At least I think that is required, you might want to double check that. Also that once created, if an op which is nested inside other ops is deleted then anything using it will become broken.
